src/pages/home/Header.tsx

```diff
@@ -7,7 +7,7 @@ export default () => (
       <img alt="useReactNYC logo" src={logo} />
       <h1>Welcome to useReactNYC</h1>
       <p className="subhead">
-        The home of New York City’s Newest and Most Exciting React Community
+        The Home of New York City’s Newest and Most Exciting React Community
```

Comment:
I feel like this should be sentence case...

/shrug

Reply:
Yeah, but it's a big title